src/sqlfluff/core/dialects/dialect_mysql.py

```diff
@@ -4,9 +4,12 @@
 https://dev.mysql.com/doc/refman/8.0/en/differences-from-ansi.html
 """
 
-from sqlfluff.core.parser import NamedSegment, OneOf, Ref
+from sqlfluff.core.parser import NamedSegment, Ref, AnyNumberOf, Sequence, OneOf
 
-from sqlfluff.core.dialects.dialect_ansi import ansi_dialect
+from sqlfluff.core.dialects.dialect_ansi import (
+    ansi_dialect,
+    CreateTableStatementSegment as ansi_CreateTableStatementSegment,
```

Comment:
Possible alternative for this:
```
import sqlfluff.core.dialects.dialect_ansi as ansi 
```

Then, later in the code, you could reference `ansi.CreateTableStatementSegment` (dot, rather than underscore). This is shorter and avoids the slight risk of typos when copying the name `CreateTableStatementSegment` -> `ansi_ CreateTableStatementSegment`.

Reply:
Totally agree. 👍